Content.Shared/Movement/Systems/SharedMoverController.Input.cs

```diff
@@ -83,17 +83,17 @@ public static MoveButtons GetNormalizedMovement(MoveButtons buttons)
             return oldMovement;
         }
 
-        protected void SetMoveInput(InputMoverComponent component, MoveButtons buttons)
+        protected void SetMoveInput(EntityUid entity, InputMoverComponent component, MoveButtons buttons)
```

Comment:
use `Entity<InputMoverComponent>` and make all the event handlers use that then just pass it directly

Reply:
Entity<T> superiority incoming

Reply:
it's disgusting but it's done